src/createRoom.ts

```diff
@@ -168,6 +168,15 @@ export default async function createRoom(opts: IOpts): Promise<string | null> {
                 },
             };
         }
+    } else {
+        createOpts.power_level_content_override = {
+            events: {
```

Comment:
This needs `...DEFAULT_EVENT_POWER_LEVELS` included iirc, because the server only does a shallow merge with the overrides